lib/CL/devices/hsa/pocl-hsa.c

```diff
@@ -892,11 +949,32 @@ pocl_hsa_run(void *dptr, _cl_command_node* cmd)
     }
 }
 
+static int run_command(char* args[])
```

Comment:
The need for this function needs plenty of comments so we remember the obscure issue later on :)